torchtune/models/clip/_transforms.py

```diff
@@ -69,6 +158,7 @@ class CLIPImageTransform:
             resolution from possible_resolutions.
             If False, it will pick the resolution that minimizes downscaling, including no downscaling at all.
             In this case, the image will only be upscaled if it's size < tile_size. Default False.
+        max_image_size (int): Used to upper-bound the input image size for export.
```

Comment:
For a regular user, i am not sure if they understand what "for export" means. Will users change this? If not, should it be a global constant in the script? Maybe if we could add a bit more context, it would solve it